### PR TITLE
Check token expiration

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [na] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

This PR corrects how the expiration dates are computed for cached Auth0 Management API tokens.

### CODE REVIEW

File: src/test/java/org/opentripplanner/middleware/auth/Auth0ConnectionTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        cachedToken = new ObjectMapper().readValue("{\"expires_in\": 86400}", TokenHolder.class);
+    }
+
+    /**
+     * Shows that TokenWrapper.getExpiresIn() is invariant
+     * (see https://github.com/auth0/auth0-java/blob/master/src/main/java/com/auth0/json/auth/TokenHolder.java)
+     * it is the duration of the token validity.
+     */
+    @Test
+    public void tokenExpiryIsActuallyTokenLifetime() throws Auth0Exception, InterruptedException {
```
Comment (br648) [inline src/test/java/org/opentripplanner/middleware/auth/Auth0ConnectionTest.java]: Auth0Exception is never thrown and can be removed.

**Review by br648 — APPROVED** [review]
Apart from one minor comment, all looks good.

File: src/test/java/org/opentripplanner/middleware/auth/Auth0ConnectionTest.java (hunk 1 above)
Comment (binh-dam-ibigroup) [reply src/test/java/org/opentripplanner/middleware/auth/Auth0ConnectionTest.java]: Test has been removed.

File: src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java [hunk 2]
```diff
@@ -147,4 +147,11 @@ private static JWTVerifier getVerifier(Request req, String token) {
     public static boolean authDisabled() {
         return hasConfigProperty("DISABLE_AUTH") && "true".equals(getConfigPropertyAsText("DISABLE_AUTH"));
     }
+
+    /**
+     * Computes a token expiration time in seconds.
+     */
+    public static long getTokenExpirationTime(TokenHolder token) {
+        return System.currentTimeMillis() + token.getExpiresIn() /* seconds */ * 1000;
+    }
```
Comment (landonreed) [inline src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java]: Looks like I missed this. Could you delete, please?

**Review by landonreed — APPROVED** [review]
LGTM, just one unused method needs deletion.

File: src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java (hunk 2 above)
Comment (binh-dam-ibigroup) [reply src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java]: Thanks for catching that. I just removed it.

**Pull request merged**